utils/plot-tasks.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+"""Script to plot tasks from profiling data.
+
+This script requires the matplotlib and numpy Python modules.
+"""
+
+from __future__ import print_function
+# mathplotlib does not support Unicode strings a column names.
+# from __future__ import unicode_literals
+
+import argparse
+import glob
+import os
+import sys
+
+from matplotlib import pyplot  # pylint: disable=import-error
+from numpy import genfromtxt  # pylint: disable=import-error
```

Comment:
genfromtxt is a function, import the module or package instead: https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports

Reply:
ack